### DISCUSSION TITLE
Styling help, has anyone figured out how to make this look modern?

### DISCUSSION THREAD

**Discussion by Jared-Dahlke** [dowjones/react-dropdown-tree-select#431] [Ideas]
Hey guys I've been messing around with custom material UI styling for this component for a couple hours now but I am no CSS expert, it would be nice if this came looking modern out of the box. Even the material UI examples on the repo have big problems, like the checkboxes are misaligned and hidden.

**Comment by mrchief** [comment]
Material UI has huge issues - their HTML emits so many extra elements purely for styling. We're not in the IE6 era anymore. :) 

If you have any framework in mind, let me know and maybe we can dabble in it together!
